### ISSUE TITLE
Continuous crash after clicking Preview Premix - Apple M1 chip

### ISSUE THREAD

**Issue by dongwe1** [sparrowwallet/sparrow#611]
Hello,
I'm not able to perform a mix because after clicking "Preview Premix" Sparrow crashes (the apple rainbow crash appears).
I tried many times, but I always obtain a crash.
I think there is a bug to fix.
Let me know if I can do something. 
Thank you.

Dongwe1

**Comment by craigraw** [comment]
Can you provide more details on the nature of the crash? For example are there any logs or crash report available?

Can you provide details on the system you are running on? Have you tried running Sparrow after a reboot without other apps running?

**Comment by dongwe1** [comment]
I run macOS Monterey 12.4 - Apple M1 iMac. 
My node (Umbrel) is connected to Sparrow. 
I tried to run Sparrow after a reboot and after have closed other apps, I also closed VPN. 
Same result: apple rainbow crash appears every time (then I have to force close Sparrow). 
I attach a screenshot. 
I don't see a Sparrow crash report. 
I see the Apple problem report that is very very long. Let me know if I you need it. Let me know if I can do something. 
Thank you.

<img width="2240" alt="Schermata 2022-07-03 alle 10 57 24" src="https://user-images.githubusercontent.com/108491829/177033382-eb1ea179-c230-4c17-b637-c8ddd5da952d.png">

**Comment by dongwe1** [comment]
I notice that in the screenshot the apple rainbow crash doesn't appear. 
![tempImageMZJav3](https://user-images.githubusercontent.com/108491829/177033481-9f10dd4e-098e-495a-8299-c86646263037.gif)

**Comment by craigraw** [comment]
Yes, I need the Apple crash report. I actually don’t know what you mean by “apple rainbow crash”. Are you talking about the spinning beach ball, or something else?

**Comment by dongwe1** [comment]
Yes, sorry for my ignorance, I mean the spinning beach ball.
I attach here the Apple crash report. I saved it in word docx, if you need another format please let me know.
Thank you.

[Sparrow_crashreport.docx](https://github.com/sparrowwallet/sparrow/files/9034647/Sparrow_crashreport.docx)

**Comment by craigraw** [comment]
Thanks for the report, marking as a duplicate of #586.
